gitpod-io/gitpod
# Adding a Button component

## Description

Adding new `<Button/>` component to normalize how we use buttons and how they appear. Still more details we can work through as we apply this to other spots using vanilla `<button>` tags, but this is a start that works for the use cases below.

Updated the following locations to use the new component, as they provided a variety of styles.
* Confirmation Modal - buttons in the footer
* Onboarding flow - Continue button
* User Settings => Profile - Update profile, and Delete account)
* Environment Variables - Create/Update modal
* Create Workspace button on Workspaces page
* New Create workspace page - Create workspace button, w/ `loading` state

## Related Issue(s)

Fixes #16441

## How to test

* Check out the places listed above on the Dashboard and verify the buttons look as expected.

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
